terraform/modules/iam_param_store/main.tf

```
@@ -15,14 +15,12 @@ resource "aws_iam_policy" "app_param_store" {
         {
             "Effect": "Allow",
             "Action": [
-                "ssm:DescribeParameters",
-                "ssm:GetParameterHistory",
-                "ssm:GetParametersByPath",
-                "ssm:GetParameters",
-                "ssm:GetParameter"
+                "secretsmanager:ListSecrets",
+                "secretsmanager:DescribeSecret",
+                "secretsmanager:GetSecretValue"
             ],
             "Resource": [
-                "arn:aws:ssm:us-east-1:${data.aws_caller_identity.current.account_id}:parameter/bb2/${var.env}/app/*"
+                "arn:aws:secretsmanager:us-east-1:${data.aws_caller_identity.current.account_id}:secret:/bb2/test/app/*"
```

@ajshred I think this be changed to work with other envs besides `test`?

Something like:
```suggestion
                "arn:aws:secretsmanager:us-east-1:${data.aws_caller_identity.current.account_id}:secret:/bb2/${var.env}/app/*"
```

---

woops!